greedybear/settings.py

```diff
@@ -37,6 +37,7 @@
     "gui.apps.GuiConfig",
     "api.apps.ApiConfig",
     "greedybear.apps.GreedyBearConfig",
+    'rest_framework',
```

Comment:
just for cleanliness, this should come before our custom apps like api and greedybear.